src/components/menu/menu.xml

```diff
@@ -7,19 +7,22 @@
         t-on-scroll="onScroll"
         t-on-wheel.stop=""
         t-on-click.stop=""
+        t-on-keydown="onKeydown"
+        tabindex="-1"
         t-on-contextmenu.prevent="">
         <t t-foreach="menuItemsAndSeparators" t-as="menuItem" t-key="menuItem_index">
           <div t-if="menuItem === 'separator'" class="o-separator"/>
           <t t-else="">
             <t t-set="isMenuRoot" t-value="isRoot(menuItem)"/>
             <t t-set="isMenuEnabled" t-value="isEnabled(menuItem)"/>
             <div
+              t-ref="{{menuItem.id}}"
               t-att-title="getName(menuItem)"
               t-att-data-name="menuItem.id"
-              t-on-click="(ev) => this.onClickMenu(menuItem, menuItem_index, ev)"
-              t-on-mouseover="(ev) => this.onMouseOver(menuItem, menuItem_index, ev)"
+              t-on-click="(ev) => this.onClickMenu(menuItem, ev)"
+              t-on-mousemove="(ev) => this.onMouseMove(menuItem, ev)"
```

Comment:
The reason to use `mousemove` here is using `mouseover` will cause bug when I select something (e.g. insert all formulas) using mouse first, and then use ArrowUp and ArrowDown to navigate the menu. I'm not sure whether I need to debounce `mousemove`.

Reply:
Yes, I tried `mouseEnter`. It has the same problem as `mouseOver` when the menu item list is too long (with scroll bar) and the menu is scrolled, because `mouseEnter` will be triggered when it enters a new menu item (div) even though the mouse doesn't move (only the scroll bar moves), whereas `mouseMove` won't.